### PR TITLE
moved book to store

### PR DESCRIPTION
**Pull request by alex311982**
Fix of a book component

### CODE REVIEW

File: bookshop/src/store/index.js [hunk 1]
```diff
@@ -40,23 +42,34 @@ const store = new Vuex.Store({
     async booklist ({commit}) {
       try {
         let {data: {data: res}} = await axios.get('getBooks')
-        commit('set', {type: 'books', items: res})
+        commit('set', {type: 'books', data: res})
```
Comment (pattt) [inline bookshop/src/store/index.js]: make it more specific. Let it be 'books'
let {data: {data: books}} = await axios.get('getBooks')
commit('set', {type: 'books', data: books})

File: bookshop/src/store/index.js [hunk 2]
```diff
@@ -40,23 +42,34 @@ const store = new Vuex.Store({
...
       } catch (e) {
 
       }
     },
+    async getBook ({commit}, {id}) {
+      try {
+        let {data: {data: res}} = await axios.get(`getBookById/${id}`)
+        commit('set', {type: 'book', data: res})
```
Comment (pattt) [inline bookshop/src/store/index.js]: call it 'book'

File: bookshop/src/store/index.js [hunk 3]
```diff
@@ -40,23 +42,34 @@ const store = new Vuex.Store({
...
+      }
+    },
+    addToCart ({ commit }) {
+      commit('addToCart')
+    },
     async genres ({commit}) {
       try {
         let {data: {data: res}} = await axios.get('getGenres')
-        commit('set', {type: 'genres', items: res})
+        commit('set', {type: 'genres', data: res})
```
Comment (pattt) [inline bookshop/src/store/index.js]: call it 'genres'

File: bookshop/src/components/Book.vue [hunk 4]
```diff
@@ -1,55 +1,50 @@
...
-  },
-  async mounted () {
-    try {
-      let {data: {data: res}} = await axios.get(`getBookById/${this.$route.params.id}`)
-      this.book = res
-    } catch (e) {
+  import { mapGetters, mapActions } from 'vuex'
 
-    }
+  export default {
```
Comment (pattt) [inline bookshop/src/components/Book.vue]: remove indents

Comment (alex311982) [reply bookshop/src/components/Book.vue]: Fixed

File: bookshop/src/store/index.js (hunk 1 above)
Comment (alex311982) [reply bookshop/src/store/index.js]: Fixed

File: bookshop/src/store/index.js (hunk 2 above)
Comment (alex311982) [reply bookshop/src/store/index.js]: Fixed

File: bookshop/src/store/index.js (hunk 3 above)
Comment (alex311982) [reply bookshop/src/store/index.js]: Fixed

**Review by pattt — APPROVED** [review]
:shipit:

**Pull request merged**